tests/casp/one_line_checks.pm

```diff
@@ -27,6 +27,8 @@ sub run() {
         # poo#16574
         # Should be replaced by actually connecting to admin node when it's implemented
         assert_script_run "grep \"master: 'dashboard-url'\" /etc/salt/minion.d/master.conf";
+        # poo#18668
```

Comment:
I prefer to see
"#Check ntp configuration on admin and worker nodes (poo#18668)" as I am too lazy to open progress each time when I read code :)